src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/migration/MigrateVideoBookingService.kt

```diff
@@ -58,9 +60,13 @@ class MigrateVideoBookingService(
       )
     }
 
-    // TODO still need to migrate if court not found
-    val court = mappingService.mapCourtCodeToCourt(bookingToMigrate.courtCode!!)
-      ?: throw NullPointerException("Court not found for code ${bookingToMigrate.courtCode} for court booking ${bookingToMigrate.videoBookingId}")
+    val court = if (bookingToMigrate.courtCode != null) {
+      mappingService.mapCourtCodeToCourt(bookingToMigrate.courtCode)
+        ?: throw NullPointerException("Court not found for code ${bookingToMigrate.courtCode} for court booking ${bookingToMigrate.videoBookingId}")
+    } else {
+      // TODO this assumes we have a court name.  Where does this go/get recorded, could potentially include/add to the comments?
```

Comment:
Thoughts?

Reply:
Yes -  looks ok. Provided we cater for an UNKNOWN court in the data.